Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -3302,3 +3302,6 @@
 
 /* Info alert message when there are Newer Releases Available */
 "You are currently running the latest release." = "You are currently running the latest release.";
+
+/* Alert button, disable the warnng when skip-show-database is set to on */
```

Comment:
Next time `en` is the only language you need to add it to. :)

Reply:
Yes, I know, but I saw that all languages have the lastest strings, so...

Reply:
It will get translated to them next time we submit automatically. :)

Reply:
👍🏼

Reply:
I know it's just in the comment, but `warnng` should be `warning`